2-mandatory/4-restaurant.js

```diff
@@ -21,7 +21,24 @@ const MENU = {
 
 let cashRegister = {
   // write code here
-}
+  orderBurger: function (balance) {
+    let { burger } = MENU;
+    if (balance > 0 && balance >= burger) {
```

Comment:
if balance >= burger than balance will always be > 0.
So can simplify to just` balance >= burger`.